azure_arc_servers_jumpstart/docs/arc_policies_.mma.md

```diff
@@ -0,0 +1,109 @@
+# Overview
+
+The following README will guide you on how to use Arc for servers to assign Azure Policies to VMs outside of Azure, wether they are on-premise or other clouds. With this feature you can now use Azure Policies to audit settings in the operating system of an Azure Arc connected servers, if a setting is not compliant you can also trigger a remediation task. 
```

Comment:
Be explicit about the fact you are doing this again a Google Cloud Platform (GCP) server instance